packages/core/src/features/cohort/availableCohortsSlice.ts

```diff
@@ -1293,12 +1423,23 @@ export const selectUnsavedCohortName = (state: CoreState): string | undefined =>
  *  -------------------------------------------------------------------------
  **/
 
+/**
+ * A hook to get the current cohort filter as a FilterSet
+ * @category Cohort
+ * @category Hooks
+ */
 export const useCurrentCohortFilters = (): FilterSet | undefined => {
   return useCoreSelector((state: CoreState) =>
     selectCurrentCohortFilterSet(state),
   );
 };
 
+/**
+ * A hook to get the current cohort filter as a FilterSet
```

Comment:
The seem the same as above. Should we specify gene and ssm here?

Reply:
It is. 

It will be removed once the refactoring for dynamic cohorts is completed. 
I updated the documentation and marked it as deprecated